cube-builder-aws/cube_builder_aws/business.py

```diff
@@ -29,14 +29,14 @@ def create_cube(self, params):
         cubes_serealized = []
         for composite_function in params['composite_function_list']:
             c_function_id = composite_function.upper()
-            cube_id = '{}_{}'.format(params['datacube'], c_function_id)
             raster_size_id = '{}-{}'.format(params['grs'], int(params['resolution']))
+            cube_id = get_cube_id(params['datacube'], c_function_id)
 
             # add cube
             if not list(filter(lambda x: x.id == cube_id, cubes)) and not list(filter(lambda x: x.id == cube_id, cubes_db)):
                 cube = Collection(
                     id=cube_id,
-                    temporal_composition_schema_id=params['temporal_schema'],
+                    temporal_composition_schema_id=params['temporal_schema'] if c_function_id.upper() != 'WARPED' else 'Anull',
```

Comment:
Why the function ``WARPED`` still used? Do you mean ``IDENTITY``?
``params['composite_function_list'] = ['WARPED', 'STK', 'MED']``